src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbnailList = document.getElementById('thumbs');
+
+thumbnailList.addEventListener('click', function(clickEvent) {
+  if (clickEvent.target.tagName === 'A' || clickEvent.target === 'IMG') {
+    clickEvent.preventDefault();
+
+    const anchorElement = clickEvent.target.tagName === 'A'
+
+      ? clickEvent.target : clickEvent.target.parentElement;
+
+    const newImgUrl = anchorElement.target.getAttribute('href');
+
+    largeImg.setAttribute('src', newImgUrl);
```

Comment:
before all the logic you need to 
`clickEvent.preventDefault();`
then you can define your target element depending on tag name, thenif your target element has class `'list-item__link'` set image src and alt